### ISSUE TITLE
Not rendering long cable and pipeline segments

### ISSUE THREAD

**Issue by Dirk--** [OpenNauticalChart/renderer#6]
https://github.com/OpenSeaMap/online_chart/issues/168

**Comment by quantenschaum** [comment]
observed this as well @ 53.6900/7.2689

**Comment by malcolmh** [comment]
This is intentional. There are may such objects that extend for 100's or 1000's of km and rendering them would kill the renderer. Mid-oceanic pipes & cables are of no navigational interest because of their depth

**Comment by quantenschaum** [comment]
>Mid-oceanic pipes & cables are of no navigational interest because of their depth

This is true, but these cables end somewhere at the coast in shallow water and there they are of navigational interest.

**Comment by malcolmh** [comment]
The work-around is to split the way a few miles from the coast where the depth is shallow enough to be a navigational hazard.

**Comment by quantenschaum** [comment]
That's a bad idea because it would duplicate the metadata attached to the line since there is no relation defined for ways in general.
